src/overlays/actors/ovl_Bg_Spot03_Taki/z_bg_spot03_taki.c

```diff
@@ -27,15 +31,141 @@ const ActorInit Bg_Spot03_Taki_InitVars = {
     (ActorFunc)BgSpot03Taki_Update,
     (ActorFunc)BgSpot03Taki_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot03_Taki/func_808ADAE0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot03_Taki/BgSpot03Taki_Init.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+extern UNK_TYPE D_06000C98;
+extern Vtx* D_06000800[];
+extern Vtx* D_06000990[];
```

Comment:
D_06000800 and D_06000990 are identical Vtx data for the waterfall. D_06000800 is buffer 0, D_06000990 is buffer 1

Reply:
So should I rename these, or is this just a remark?

Reply:
I'd just add a comment, since we are still figuring out a naming scheme for object file variables